apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -479,7 +479,7 @@ defmodule OMG.Eth.RootChain do
 
   def decode_in_flight_exit_output_finalized(log) do
     non_indexed_keys = [:in_flight_exit_id, :output_index]
-    non_indexed_key_types = [{:uint, 192}, {:uint, 8}]
+    non_indexed_key_types = [{:uint, 160}, {:uint, 8}]
```

Comment:
where in solidity did you find the change from uint 160 to 192?

Reply:
(I hope I got this right, since IFEs aren't really tackled yet - separate issue).

This one I actually was just aware of (was following the debate on the topic of the change 192->160). But you can also see it in the diffs, wherever `exit_id`/`exitId` is mentioned.